instagram-app/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import './CommentSection.css'
+import Comment from './Comment'
+
+function CommentSection(props){
+    return(
+        <div className="comment-section">
+            <p> *heart and comment icons*</p>
```

Comment:
Good placeholder. Don't forget to grab the images. Try fontawesome, works well with React.